include/vapor/ParamsMgr.h

```diff
@@ -432,6 +432,29 @@ class PARAMS_API ParamsMgr : public MyBase {
 	return(itr != _otherRenParams.cend() ? itr->second->GetParams(classType) : NULL);
  }
 
+ //! Optain any render paramers registered by the application for a given data set
+ //!
+ //! This method returns params that have been registered on the
+ //! ParamsMgr via RegisterAppParams() for the data set named
+ //! by \p dataSetName;
+ //!
+ //! \param[in] dataSetName  
+ //! \param[out] appRenderParams a vector of application render params associated with \p dataSetName  
+ //!
+ //! \sa ParamsMgr()
+ //
+ void GetAppRenderParams(string dataSetName, vector <RenderParams *> &appRenderParams) const {
```

Comment:
Is it part of the c++ programming convention to place function implementations in the cpp file, instead of in the header ?

This particular header is widely included, meaning that it is repeatedly compiled many times throughout VAPOR.

Reply:
Done